calendar.md

```diff
@@ -0,0 +1,29 @@
+---
+permalink: /schedule
+layout: calendar
+title: Schedule
+styles:
+  - url: https://unpkg.com/@fullcalendar/core@4.3.0/main.min.css
+    hash: sha384-UYbBlSMkHrbuUVqSs26Rm1UEii5VOTR80mD2wjrJaIedgHgS5LWDt9d7rcbqEDxR
+  - url: https://unpkg.com/@fullcalendar/daygrid@4.3.0/main.min.css
+    hash: sha384-3/CmTldIZRXuPMPFO0YHhiLwdsqbsB8YEG+psHdxNbnNBTrci0MOdt/hbPRVmtS8
+  - url: https://unpkg.com/@fullcalendar/timegrid@4.3.0/main.min.css
+    hash: sha384-Y2E7KGXSxkviA14jxA5DeBmVwovKH3ipCOXJ1ByetT9DLlwehyYf/3Lm2wSncMR2
+  - url: https://unpkg.com/@fullcalendar/list@4.3.0/main.min.css
+    hash: sha384-I6BmgTJPPB58p0uMGaoLY/wFTFFlO4VSmjQYBfNzCzejCImTYNj/yXYmKBeUjtHH
+  - url: https://fonts.googleapis.com/icon?family=Material+Icons
+scripts:
+  - url: https://unpkg.com/@fullcalendar/core@4.3.0/main.min.js
+    hash: sha384-rouc52Uk/cLTKcEeTGcW5Jg76vjIdbKE/Zd3U47ZMc4ezmiCtKCRuT7TRd/ai7f1
+  - url: https://unpkg.com/@fullcalendar/daygrid@4.3.0/main.min.js
+    hash: sha384-Vg8WM1KZn59u30ahJ4mGZwEw2R4ieRhhvX2PPiKKCSpuxms+HvoC3XPii6CnMvFD
+  - url: https://unpkg.com/@fullcalendar/timegrid@4.3.0/main.min.js
+    hash: sha384-yyTZuDPNQFp/xKe4Yk67GZBC6Pm3OXdipzs78KRRA37YWJuy3rcgDQTxCxbIviHs
+  - url: https://unpkg.com/@fullcalendar/list@4.3.0/main.min.js
+    hash: sha384-fG3/vSufb+AA91JKrVZZiS16J72wgK0u/CK+afw3Kw2tbVEPEFcTHac7sJabZd31
+  - url: https://unpkg.com/@fullcalendar/google-calendar@4.3.0/main.min.js
+    hash: sha384-Ha5I8QPCC/z63Vv2VZG8QZIXGIhDPrLLu3fMx46RG+FLQqpWNzsA8mZIR9YXPahZ
+  - url: https://unpkg.com/@fullcalendar/bootstrap/main.min.js
+    hash: sha384-iE2vNnlZ9zLj/SJ+wylWYZLKMYF/03QtrLFdByV59XHKHCie9UTGNJnH6imUYv17
```

Comment:
bootstrap hasn't been versioned like all the other imports so unpkg rewrites it to [https://unpkg.com/@fullcalendar/bootstrap@5.1.0/main.min.js](https://unpkg.com/@fullcalendar/bootstrap@5.1.0/main.min.js) which 404s
```suggestion
  - url: https://unpkg.com/@fullcalendar/bootstrap@4.3.0/main.min.js
    hash: sha384-xcJfdzrkJuIsmAm2DIzmmFsRJA9pItoEHjpES5SO9uQJ8cL/fN+ARTFFPpsB8utm
```

Athough interestingly it does still seem to be working, since removing it breaks the page.